src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1265,6 +1284,39 @@ class ApplicationManagerImpl
    */
   void SendMobileMessage(smart_objects::SmartObjectSPtr message);
 
+  /*
+   * @brief Sets default value of the HELPPROMT global property
+   * to the first vrCommand of each Command Menu registered in application
+   *
+   * @param app Registered application
+   * @param is_timeout_promp Flag indicating that timeout prompt
+   * should be reset
+   *
+   * @return TRUE on success, otherwise FALSE
+   */
+  bool ResetHelpPromt(ApplicationSharedPtr app);
```

Comment:
@mked-luxoft why the function is not const?

Reply:
@LuxoftAKutsan fixed in be00949b515b17d868e96968b8c7729b4fee9d3d